tutor/interactive.py

```diff
@@ -41,6 +41,7 @@ def ask_questions(config: Config, defaults: Config) -> None:
         dev_values: Config = {
             "LMS_HOST": "local.overhang.io",
             "CMS_HOST": "studio.local.overhang.io",
+            "PREVIEW_LMS_HOST": "preview.local.overhang.io",
```

Comment:
I would rather avoid adding an extra question to the quickstart. I think that the default value is reasonable for most people, and we don't need to ask everyone about it.